apps/ui/src/i18n/locale/en/enSection.json

```diff
@@ -71,6 +71,8 @@
 		"title": "Collaborators",
 		"description1": "Please include the names of all investigators, collaborators, research staff (including post-docs) and students (including graduate students), who will have access to the PCGL Controlled Data in order to work on the Research Summary as outlined in Section D of this application",
 		"note": "* Please note: co-investigators, collaborators or students at other institutions should not be included in this list. They will have to submit a separate application for access to controlled data.",
+		"deleteModalTitle": "Are you sure you want to delete collaborator?",
+		"deleteModalDes": "You are about to delete collaborator, {{name}} from application PCGL-{{appId}}",
```

Comment:
This is nitpicky, but let's use `deleteModalDescription` for consistency w/ the rest of the file, could possibly cause confusion if someone is searching for the 'description' value only to realize it's a 'Des' value